client/extensions/woocommerce/state/sites/request.js

```diff
@@ -31,6 +31,7 @@ const _request = ( method, path, siteId, body ) => {
 		{
 			path: `/wc/v3/${ path }&_method=${ method }`,
 			body: body && JSON.stringify( body ),
+			json: true,
```

Comment:
Yeah, I get this response:
`{"code":500,"headers":[{"name":"Content-Type","value":"application\/json"}],"body":{"error":"no_response_body","message":"Server could not read response."}}`

But adding `json: true` fixes it. However, it appears adding `json: true` has broken a few tests somehow. Not sure why that would be.